templates/user.twig

```diff
@@ -13,7 +13,7 @@
 
 	{% block content %}
 <div class="row" style="margin-top:-150px;">
-<p class="text-white">are you fucking kidding me guys this is literally just the poktube semantic ui profile layout but without the user infomation detail bullshit</p>
+<p class="text-white">{{ __("are you fucking kidding me guys this is literally just the poktube semantic ui profile layout but without the user infomation detail bullshit") }}</p>
```

Comment:
why do we need to localize placeholder text?

Reply:
why not

Reply:
PLACEHOLDER TEXT IS PLACEHOLDER TEXT, NO ONE IS GOING  TO SEE IT IN THE FINAL PRODUCT  AND THUS DOES NOT  NEED TO BE LOCALIZED